lib/bloc/backup/backup_bloc.dart

```diff
@@ -23,8 +23,11 @@ import '../async_action.dart';
 import 'backup_actions.dart';
 
 class BackupBloc {
-  static const String _signInFailedCode = "AuthError";
-  static const String _notFoundCode = "NotFoundError";
+  static const String _signInFailedCode = "401";
+  static const String _methodNotFound = "405";
+  static const String _notFoundCode = "404";
+  static const String _noAccess = "403";
+  static const String _resultError = "ResultError";
```

Comment:
Is ResultError code of the error?

Reply:
Yes, this was wrongly added as the ResultError is the code for the 404 errormessage. Will remove